### PR TITLE
Add EU entities

### PR DESCRIPTION
**Pull request by samsamann**

### CODE REVIEW

File: src/main/java/com/github/bfh/study/slb/provider/eu/entities/PassportInfo.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.github.bfh.study.slb.provider.eu.entities;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+public class PassportInfo extends SubEntry {
```
Comment (ghost) [inline src/main/java/com/github/bfh/study/slb/provider/eu/entities/PassportInfo.java:8]: @samsamann, there is at the moment no use for this class... Where should this class be used?

File: src/main/java/com/github/bfh/study/slb/provider/eu/entities/CitizenInfo.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.github.bfh.study.slb.provider.eu.entities;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+public class CitizenInfo extends SubEntry {
```
Comment (ghost) [inline src/main/java/com/github/bfh/study/slb/provider/eu/entities/CitizenInfo.java:8]: @samsamann, this class has no usage...

File: src/test/java/com/github/bfh/study/slb/imports/parser/PartialXmlParserTest.java [hunk 3]
```diff
@@ -56,4 +59,20 @@ public void testParserWithoutElements() throws Exception {
         parser.close();
         assert count == 0 : "the count value should be zero";
     }
+
+    public void testEuListParsing() throws Exception {
```
Comment (ghost) [inline src/test/java/com/github/bfh/study/slb/imports/parser/PartialXmlParserTest.java:63]: @samsamann, why do you don't use the context? and the EuSanctionsList.java as in the method testParsing()?

Comment (samsamann) [reply src/test/java/com/github/bfh/study/slb/imports/parser/PartialXmlParserTest.java:63]: testParsing() tests only if the JAXB annotations are correct. testEuListParsing is an integration test and uses the original xml file.

Comment (ghost) [reply src/test/java/com/github/bfh/study/slb/imports/parser/PartialXmlParserTest.java:63]: OK

**Comment by samsamann** [comment]
@joeggu pls have a look again! I added the missing elements

**Review by ghost — APPROVED** [review]
Looks pretty! Well Done ;-)

**Pull request merged**